src/compiler/utilities.ts

```diff
@@ -3696,6 +3696,7 @@ namespace ts {
             getDiagnostics,
         };
 
+        function lookup(diagnostic: DiagnosticWithLocation): DiagnosticWithLocation | undefined;

```

Comment:
It doesn't seem like you intended to make a non-overloaded function here?

Reply:
Yeah, that was a mistake, finally resolved it.